src/axom/core/Macros.hpp

```diff
@@ -28,9 +28,11 @@
 #if defined(__CUDACC__)
 #define AXOM_DEVICE __device__
 #define AXOM_HOST_DEVICE __host__ __device__
+#define AXOM_HOST __host__
```

Comment:
This is redundant, since not having a decorator is effectively a `HOST` functions.

Reply:
`AXOM_HOST` doesn't otherwise appear to be defined.  
See the definition of `AXOM_HOST_LAMBDA` below.

Are you suggesting removing the `__host__`  part?
Or removing `AXOM_HOST` from the CUDA part of `AXOM_HOST_LAMBDA` ?

Reply:
Here is the code I was referring to (line 53):

https://github.com/LLNL/axom/blob/889cf6c8ca31ac5b2f6d9e3a63c065c1b754cb32/src/axom/core/Macros.hpp#L50-L58

Reply:
I see (who did this? 😄 ) -- So, my preference would be to remove `AXOM_HOST` from the `AXOM_HOST_LAMBDA` definition. But, honestly either way is fine.

Reply:
Thanks @gzagaris.

I'm going to keep this as is for this PR since I've already tested in with our host-configs on LC and being more explicit in this case can't hurt. 

We can always remove it in the future if that is a better approach.